src/main/java/com/rabbitmq/jms/client/RMQSession.java

```diff
@@ -869,7 +869,7 @@ private void declareRMQQueue(RMQDestination dest, String queueNameOverride, bool
     @Override
     public Topic createTopic(String topicName) throws JMSException {
         illegalStateExceptionIfClosed();
-        RMQDestination dest = new RMQDestination(topicName, false, false);
+        RMQDestination dest = new RMQDestination(topicName, topicName, "");
```

Comment:
We cannot change the default behavior like this, it will break existing applications. Note CI failed to pass the tests, there are regression.